src/js/common/helpers/login-helpers.js

```diff
@@ -46,6 +46,8 @@ export function getUserData() {
       commonStore.dispatch(updateProfileField('userFullName.last', userData.last_name));
       commonStore.dispatch(updateProfileField('gender', userData.gender));
       commonStore.dispatch(updateProfileField('dob', userData.birth_date));
+      commonStore.dispatch(updateProfileField('status', json.data.attributes.va_profile.status));
```

Comment:
Probably not something to do right now, but seems like this should be refactored to use one action. This is probably triggering a bunch of unnecessary React renders.